src/Translation_unit.ml

```diff
@@ -130,7 +130,7 @@ let parse_print (XUnit xunit) (conf: Conf.t) iname ifile ic ofile =
   in
   let source = In_channel.with_file ifile ~f:In_channel.input_all in
   try parse_print_ 1 source ifile ic with
```

Comment:
Could you change `try` to `try[@warning "-28"]` to suppress the warning when building with 4.06.

It would also be good to reference https://github.com/ocaml-ppx/ocaml-migrate-parsetree/issues/34 so that if/when the type of exceptions is migrated by migrate-parsetree, then we can update and remove the warning suppression.

Reply:
Definitely - I'll update this PR with the annotation and reference to the omp issue.